test/core/transport/chttp2/too_many_pings_test.cc

```diff
@@ -364,10 +372,14 @@ TEST_F(KeepaliveThrottlingTest, KeepaliveThrottlingMultipleChannels) {
           const_cast<char*>(GRPC_ARG_HTTP2_BDP_PROBE), 0)};
   grpc_channel_args client_channel_args = {GPR_ARRAY_SIZE(client_args),
                                            client_args};
-  grpc_channel* channel = grpc_insecure_channel_create(
-      server_address.c_str(), &client_channel_args, nullptr);
-  grpc_channel* channel_dup = grpc_insecure_channel_create(
-      server_address.c_str(), &client_channel_args, nullptr);
+  grpc_channel_credentials* creds = grpc_insecure_credentials_create();
+  grpc_channel* channel =
+      grpc_channel_create(server_address.c_str(), creds, &client_channel_args);
+  grpc_channel_credentials_release(creds);
+  grpc_channel_credentials* another_creds = grpc_insecure_credentials_create();
```

Comment:
No need for `another_creds`. It's fine to use the same `creds` object for both channels.

Reply:
Ah right. Done!